app/src/main/java/com/sun_asterisk/youtubebackground/screen/home/HomeAdapter.java

```diff
@@ -0,0 +1,65 @@
+package com.sun_asterisk.youtubebackground.screen.home;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+import com.bumptech.glide.Glide;
+import com.sun_asterisk.youtubebackground.R;
+import com.sun_asterisk.youtubebackground.data.model.Video;
+import java.util.ArrayList;
+import java.util.List;
+
+public class HomeAdapter extends RecyclerView.Adapter<HomeAdapter.ViewHolder> {
+    private Context mContext;
+    private List<Video> mVideos;
+
+    public HomeAdapter(Context context) {
+        mContext = context;
+        mVideos = new ArrayList<>();
+    }
+
+    @NonNull
+    @Override
+    public ViewHolder onCreateViewHolder(@NonNull ViewGroup viewGroup, int i) {
+        return new ViewHolder(LayoutInflater.from(mContext).inflate(R.layout.video_item, viewGroup, false));
+    }
+
+    @Override
+    public void onBindViewHolder(@NonNull final ViewHolder viewHolder, int i) {
+        Video video = mVideos.get(i);
```

Comment:
tạo 1 method riêng để xử lý view e (bindView)
```suggestion
      viewHolder.bindView(mVideos.get(postion))
```